k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
@drewwills The docker registry is to docker, as maven centeral is to Java.
yes, we can run without it, but using the registry would allow uPortal to distribute pre-built images.